src/Toml/Type/Printer.hs

```diff
@@ -132,18 +148,22 @@ prettyKeyValue options i = mapOrdered (\kv -> [kvText kv]) options . HashMap.toL
   where
     kvText :: (Key, AnyValue) -> Text
     kvText (k, AnyValue v) =
-      tabWith options i <> prettyKey k <>  " = " <> valText v
-
-    valText :: Value t -> Text
-    valText (Bool b)    = Text.toLower $ showText b
-    valText (Integer n) = showText n
-    valText (Double d)  = showDouble d
-    valText (Text s)    = showText s
-    valText (Zoned z)   = showZonedTime z
-    valText (Local l)   = showText l
-    valText (Day d)     = showText d
-    valText (Hours h)   = showText h
-    valText (Array a)   = "[" <> Text.intercalate ", " (map valText a) <> "]"
+      let lKey = Text.length $ prettyKey k
+          lTab = Text.length $ tabWith options i
+          off = lKey + lTab + 1
```

Comment:
I don't think that we should align on the name.
It is enough to have the same indentation as the specified in options 🙂

Reply:
Let imagine long key name, how do you see it looks better?
1.
```
longname = [ "text"
, "text2"
, "text3"
]
```
2.
```
longname = 
[ "text"
, "text2"
, "text3"
]
```
or otherwise?